message/messages.go

```diff
@@ -49,12 +49,14 @@ type InboundMessage interface {
 	// BytesSavedCompression returns the number of bytes that this message saved
 	// due to being compressed
 	BytesSavedCompression() int
+	// Returns string representation of message
+	String() string
```

Comment:
We should embed `fmt.Stringer` at the top of this interface rather than re-specifying the method signature.